apps/server/src/controllers/v1/events.ts

```diff
@@ -130,6 +131,22 @@ router.put('/:id/comments/:commentId', verifyToken, verifyCommentAuthor, async (
     res.sendStatus(204)
 })
 
+router.delete('/:id/comments/:commentId', verifyToken, verifyCommentAuthor, async (req: Request, res: Response) => {
+    const session = await mongoose.startSession()
+    try {
+        session.startTransaction()
+        await CommentsModel.deleteOne({ _id: req.params.commentId })
+        await EventsModel.updateOne({ _id: req.params.id }, { $pull: { comments: new mongoose.Types.ObjectId(req.params.commentId) } })
```

Comment:
이런식으로 session 값을 넘겨줘야 할거에요. 
```suggestion
        await CommentsModel.deleteOne({ _id: req.params.commentId }, { session })
        await EventsModel.updateOne({ _id: req.params.id }, { $pull: { comments: new mongoose.Types.ObjectId(req.params.commentId) } }, { session })
```

Reply:
그렇네요 ㅠㅠ 다음에는 잘 찾아보고 반영하도록 하겠습니다!